build_docs.py

```diff
@@ -768,13 +707,19 @@ def build(self):
         if self.language.tag == "ja":
             # Since luatex doesn't support \ufffd, replace \ufffd with '?'.
             # https://gist.github.com/zr-tex8r/e0931df922f38fbb67634f05dfdaf66b
-            # Luatex already fixed this issue, so we can remove this once Texlive is updated.
+            # Luatex already fixed this issue, so we can remove this once Texlive
+            # is updated.
             # (https://github.com/TeX-Live/luatex/commit/eaa95ce0a141eaf7a02)
```

Comment:
Not really related to this pull request but the commit sha seems invalid.

Reply:
Oh that's right! @atsuoishimoto any idea what this was?

Reply:
Hi,
I believe the commit is https://github.com/TeX-Live/luatex/commit/af5faf1 now. Should I make an another PR for this?

Reply:
Thanks @atsuoishimoto, no worries i'll push this fix in this PR, it's not like there's a lot of conflicts around here :D